src/routes/+layout.svelte

```diff
@@ -27,6 +28,9 @@
 			dataIndexStore.set(response.data);
 		}
 	});
+	if ($applicationState.isAuthenticated && $userProfile.account.uid < 10000) {
+		goto('/settings/firstlogin');
+	}
```

Comment:
You need to check for this after checking authentication state over websocket instead of checking in layout.